### PR TITLE
Small fixes from #724

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

File: assets/javascripts/shapes.js [hunk 1]
```diff
@@ -344,15 +344,17 @@ CanvasState.prototype.draw = function() {
...
-	  ctx.strokeRect(mySel.x - lineWidth, mySel.y - lineWidth,
-			 mySel.w + lineWidth * 2, mySel.h + lineWidth * 2);
-      }
+    if (this.selection != null) {
+      var lineWidth = 1;
+      ctx.strokeStyle = 'white';
+      ctx.lineWidth = lineWidth;
+      // plainly ignore in phantomjs
+      if (typeof ctx.setLineDash === "function")
+	ctx.setLineDash([10, 15]);
```
Comment (okurz) [inline assets/javascripts/shapes.js]: indention looks weird here

**Comment by okurz** [comment]
I don't understand the PR title but besides this and the weird indention this is good to go, fix the indention and auto-LGTM

File: assets/javascripts/shapes.js (hunk 1 above)
Comment (coolo) [inline assets/javascripts/shapes.js]: yeah, I switched back to what everyone elses (mainly @ancorgs): 2 spaces

**Comment by coolo** [comment]
I cherry picked the commits in #724 into a branch called pick-simple-ones - and github creates titles from that

**Pull request merged**